sdl_android_lib/src/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -53,6 +54,7 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 import android.os.RemoteException;
+import android.support.v4.content.ContextCompat;
```

Comment:
We don't want to use any of the support libraries. That would force our developers to use the same support library. I suggest trying to use the normal context method if android version is 23+, or fully qualified contextcompat with a try catch